src/main/frontend/app/components/ReviewListContainer.js

```diff
@@ -1,7 +1,10 @@
-import React from "react"
+import React, { useState } from "react"
 import ReviewInfo from "./ReviewInfo"
 
 const ReviewListContainer = (props) => {
+  const [deleted, setDeleted] = useState(false)
+
+  console.log(deleted)
```

Comment:
Does this console.log have a purpose in production? or is is solely for testing? If the later then it should be removed.